diff --git a/index.d.mts b/index.d.mts
index a8f043b4160fc72d25f9e43c694191aa53dcff9b..100d5da880fcbf4ff1dc4f5b582be60c6b2b940e 100644
--- a/index.d.mts
+++ b/index.d.mts
@@ -3084,7 +3084,7 @@ declare function useMediaControls(target: MaybeRef<HTMLMediaElement | null | und
             };
             readonly length: number;
             getCueById: (id: string) => TextTrackCue | null;
-            [Symbol.iterator]: () => ArrayIterator<TextTrackCue>;
+            [Symbol.iterator]: () => IterableIterator<TextTrackCue>;
         } | null;
         activeCues: {
             [x: number]: {
@@ -3134,7 +3134,7 @@ declare function useMediaControls(target: MaybeRef<HTMLMediaElement | null | und
             };
             readonly length: number;
             getCueById: (id: string) => TextTrackCue | null;
-            [Symbol.iterator]: () => ArrayIterator<TextTrackCue>;
+            [Symbol.iterator]: () => IterableIterator<TextTrackCue>;
         } | null;
     }[], UseMediaTextTrack[] | {
         id: number;
@@ -3191,7 +3191,7 @@ declare function useMediaControls(target: MaybeRef<HTMLMediaElement | null | und
             };
             readonly length: number;
             getCueById: (id: string) => TextTrackCue | null;
-            [Symbol.iterator]: () => ArrayIterator<TextTrackCue>;
+            [Symbol.iterator]: () => IterableIterator<TextTrackCue>;
         } | null;
         activeCues: {
             [x: number]: {
@@ -3241,7 +3241,7 @@ declare function useMediaControls(target: MaybeRef<HTMLMediaElement | null | und
             };
             readonly length: number;
             getCueById: (id: string) => TextTrackCue | null;
-            [Symbol.iterator]: () => ArrayIterator<TextTrackCue>;
+            [Symbol.iterator]: () => IterableIterator<TextTrackCue>;
         } | null;
     }[]>;
     selectedTrack: vue.Ref<number, number>;
