diff --git a/dist/index.cjs b/dist/index.cjs
index 754e3ac52c21f200cc8520bf77337947c57e7923..fd6e01f6376c2bdfa9a6693d191b1ec9cc4f1936 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -489,6 +489,7 @@ const file = defineGenerator({
   async generate({ args, config, url }) {
     const fullPath = resolvePath(args.src, { url, dir: config.dir });
     let contents = await promises.readFile(fullPath, "utf8");
+    contents = contents.trim();
     if (args.code) {
       contents = mdbox.md.codeBlock(
         contents,
diff --git a/dist/index.mjs b/dist/index.mjs
index 5a800f1256f2ca858bd64a8e6988a86f82429064..fbd3d61f5fae9e9fc1735ab3c83f034c854fe790 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -483,6 +483,7 @@ const file = defineGenerator({
   async generate({ args, config, url }) {
     const fullPath = resolvePath(args.src, { url, dir: config.dir });
     let contents = await readFile(fullPath, "utf8");
+    contents = contents.trim();
     if (args.code) {
       contents = md.codeBlock(
         contents,
